src/components/kytos/inputs/Dropdown.vue

```diff
@@ -70,6 +70,14 @@ export default {
       }
       this.$emit('update:value', this.selected)
       this.action(this.selected)
+    },
+    clear () {
+      this.selected = '';
+      if(this.options && this.options.length > 0) {
+        this.options.forEach((item) => {
+          if (this.selected == '' && item.selected) {this.selected = item.value}
```

Comment:
@rmotitsuki, If this.options isn't empty and an item is selected there, wouldn't this set again `this.section = item.value` from the options? I wonder if this is the behavior we need here, since after a clear it's supposed to be empty. Unless `item.selected` is already bidirectionally bound and this will never execute, but if it never branches here then we might not need it. If you could clarify that'd be great, I'm a bit confused this `this.select` can be set again on this `forEach`